src/client/results/journalism/entities/PlayerInformation.js

```diff
@@ -1,9 +1,134 @@
+        <tbody>
+          <tr>
+            <td colSpan="2">
+              <div
+                className="ui medium centered bordered rounded image"
+                style={{maxHeight: "300px", width:"100%", marginLeft:"auto", marginRight:"auto", overflow:"hidden"}}
+              >
+                <img
+                  src={tryPropertyOrElse(desc, "thumbnail", "none")}
+                  style={{width:"100%", marginLeft:"auto", marginRight:"auto"}}
+                />
+              </div>
+            </td>
+          </tr>
+          <tr>
+            <td><strong>Full name:</strong></td>
+            <td><span property="http://dbpedia.org/property/fullname">{tryPropertyOrNA(desc, "fullname")}</span></td>
+          </tr>
+          <tr>
+            <td><strong>Birth date:</strong></td>
+            <td><span property="http://dbpedia.org/property/birthDate">{tryPropertyOrNA(desc, "birthdate")}</span> ({age} years old)</td>
+          </tr>
+          <tr>
+            <td><strong>Nationality:</strong></td>
+            <td><i className={`${details.playerNationality} flag`} /><span property="https://schema.org/nationality">{details.nationality}</span></td>
+          </tr>
+          <tr>
+            <td><strong>Current team:</strong></td>
+            <td><a href={tryPropertyOrNA(desc, "currentclub")} property="http://dbpedia.org/property/currentclub">{tryPropertyOrNA(desc, "currentclubname")}</a></td>
+          </tr>
+          <tr>
+            <td><strong>Position:</strong></td>
+            <td><a href={tryPropertyOrNA(desc, "position")} property="http://dbpedia.org/ontology/position">{tryPropertyOrNA(desc, "positionlabel")}</a></td>
+          </tr>
+          <tr>
+            <td><strong>Contracted until:</strong></td>
+            <td><abbr title="From http://api.football-data.org/"><span property="https://schema.org/endDate">{details.contractUntil}</span></abbr></td>
+          </tr>
+          <tr>
+            <td><strong>Market value:</strong></td>
+            <td typeof="https://schema.org/MonetaryAmount"><abbr title="From http://api.football-data.org/"><span property="https://schema.org/value">{details.marketValue}</span></abbr></td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  )
+};
+
+const PastTeams = ({teams}) => (
   <div>
-    Player Info
+    <h2 class="ui header">Past teams</h2>
+    <div className="ui relaxed divided list">
+      {teams.map((team) => (
+        <div className="item" typeof="http://dbpedia.org/ontology/team">
+          <i className="large soccer middle aligned icon"></i>
+          <div className="content">
+            <a className="header" href={tryPropertyOrNA(team, "team")} property="rdfs:label">{tryPropertyOrNA(team, "teamname")}</a>
+            <div className="description" property="rdfs:comment">{tryPropertyOrNA(team, "teamcomment")}</div>
+          </div>
+        </div>
+      ))}
+    </div>
   </div>
-);
+)
+
+const PlayerInformation = ({playerInfo}) => {
```

Comment:
Minor thing, if you make changes, can you put playerInformation at the top of the file since it's the top most component